zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -967,6 +975,17 @@ def test_003_object_storage_and_secondary_block(self):
         content = primary_object_one.get()['Body'].read().decode('UTF-8')
         self.assertEqual(content, obj_data)
 
+        if self.virtual_hosted_bucket_enabled:
+            # test virtual hosted bucket
+            logging.info('Testing virtual hosted bucket')
+            primary_client.Bucket(container_name).Acl().put(ACL='public-read')
+            primary_client.Object(container_name,obj_name).Acl().put(ACL='public-read')
```

Comment:
One doubt around this. The above makes the test container publically available. Later on the test checks access for the test container using test credentials. Given that the container is now publically available, would the test credentials even be considered then?

I feel it would be preferable to isolate this test into a test method of it's own (and not share a container name with the other tests), also given that this test method is awfully long already

Reply:
This test is moved to test_005_virtual_hosted_bucket function